rdrf/rdrf/settings.py

```diff
@@ -357,8 +357,10 @@
         },
     },
     'loggers': {
-        'django': {
+        '': {
             'handlers': ['console', 'file'],
+            'level': 'DEBUG' if DEBUG else 'INFO',
+            'propagate': True
         },
         'django.request': {
```

Comment:
Can we make this '' instead of 'django.request' to mail all CRITICAL logs to admins?